src/App.js

```diff
@@ -74,7 +74,7 @@ function App() {
   };
 
   // TODO: Remove this when all the new navigation features are available
-  const header = localStorage.getItem('feature-flag:navstyle') === 'DUOS-1668' //eslint-disable-line
+  const header = env === 'dev'
```

Comment:
We're going to be enabling this on everything but prod, so you'll need to say `env !== 'prod'`

Reply:
You'll also have to update the test in `auth.cy.js` so that it checks on the new console page.

Reply:
Looking at the test in `auth.cy.js`, I don't actually think it needs to change. The text that they're looking for, "Admin Console" should actually still be visible in the new header - Is that right? Am I understanding thsi correctly?

Reply:
The current test is running on the mobile view as seen in the screenshot below.

![Screen Shot 2022-06-29 at 10 31 31 AM](https://user-images.githubusercontent.com/6664778/176463125-19e5d5bb-aabc-4a1c-9ad2-07d24edb33d8.png)

Because of that, the 'Admin Console' label on the navbar doesn't show up. This means you have a couple of ways to approach updating the test. You could either update the test so that it checks on the desktop view or you can update the `cy.contains` statement to look for something else on the page.

Reply:
Unfortunately, there's no other text on that page that reallyyyyy indicates the page is part of the admin console... Ok, thanks for working with me on this! ^_^ I'll try forcing the desktop view

Reply:
Yeah the integration tests are 1) old and 2) extremely minimal.  Feel free to make the tests more detailed so that anyone can look at the test and know for sure that it's testing for stuff that's uniquely admin.